lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js

```diff
@@ -54,48 +43,23 @@ module.exports = CustomElementsPropertiesActivator;
 
 
 
-// helpers ////////////////////////////////////
-
+// helpers //////////
 
 var CUSTOM_PROPERTIES_PATTERN = /^custom-/;
 
-var DEFAULT_ENTRIES_VISIBLE = {
-  _all: false,
-  id: true,
-  name: true
-};
-
 function isCustomEntry(entry) {
   return CUSTOM_PROPERTIES_PATTERN.test(entry.id);
 }
 
-function isEntryVisible(entry, template) {
-
-  var entryId = entry.id;
-
-  if (entryId === 'elementTemplate-chooser' || isCustomEntry(entry)) {
-    return true;
-  }
-
-  var entriesVisible = template.entriesVisible || DEFAULT_ENTRIES_VISIBLE;
-
-  if (typeof entriesVisible === 'boolean') {
-    return entriesVisible;
+function isEntryVisible(element, entry, group, tab, template) {
```

Comment:
Lot's of unused arguments there. Are they still needed to prevent breaking changes?

Reply:
I've tried to streamline the events a bit. In this case, I only need the tab but someone might need the entry, the group or the tab to determine if an entry should be visible. I'll remove the unused parameters from this function but they will stay in the method that calls it.

Reply:
Breaking changes will be introduced by firing both `propertiesPanel.isEntryVisible` and `propertiesPanel.isEntryEditable` with more information.